### ISSUE TITLE
[ACCESSIBILITY] Invisible back arrow

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#5458]
1. Take the Health Screener
2. Fill out the answers: Sore throat, over 14, below 100.5, No, and No
3. Submit your answers
4. Tap on the "Get a Cold Pack"
5. See how the link opens in the app without a visible back arrow (it is black on navy)
6. Tap on where the back arrow would display and see how it is functional

Please replace the black back arrow with a white the back chevron icon.

7.3.13 (Android, dev)

![Screenshot_20251019-133900.png](https://github.com/user-attachments/assets/d176822e-430c-41cf-9497-c133be7b3c96)
